src/components/Main/Article-By-Id.jsx

```diff
@@ -0,0 +1,45 @@
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import { getArticleById } from "../utils/Api-Util";
+
+function ArticleById(params) {
+  const { articleid } = useParams();
+  const [individualArticle, setIndividualArticle] = useState([]);
```

Comment:
As its only one article, makes more sense for this to be an object, not an array